enhancements/network/service-egress-traffic-steering.md

```diff
@@ -262,8 +340,16 @@ To make it work for our use-case we'd have to refactor a lot of its functionalit
 Having said that, the solution proposed here will probably share/reuse some of the code of EgressIP as they have some sort of similarity.
 ## Design Details
 Since both EgressIP and this feature create logical router policies on the cluster router, the policies created here will use a higher priority than the EgressIP ones.
-This means that if a pod is an endpoint of both an EgressIP and a LoadBalancer service annotated with a valid `k8s.ovn.org/egress-service` annotation the service's ingress IP will be used for the SNAT.
+This means that if a pod is an endpoint of both an EgressIP and an EgressService the service's ingress IP will be used for the SNAT.
+
+An option to have the CRD cluster-scoped/namespaced that leverages selectors to match the services was considered, however the current 1-1 mapping between EgressServices and LoadBalancer services design was picked as it offers some advantages over selectors:
+- It matches the old annotation way the most, presenting a clear togglable mechanism.
+
+- Simplicity of configuration - from a user's perspective a 1-1 mapping makes it clear which service should act as an "Egress Service" and what node is in charge of its traffic. When looking at a resource in charge of handling multiple services it would not be clear which services should be configured and it would need to have a long status field mapping each service to its node.
```

Comment:
I don't think it would be a mapping of service to node right? It would be a single node, so the EgressService would be like:
```
Status:
  Host: nodeA
  Services:
     - svcA
     - svcB
     - svcC
```

Reply:
that would mean that a single node is in charge of all the services in a given resource, which is probably something we would not like to do (we should distribute services between many nodes)
it's more similar to:
```
Status:
  - svcA: nodeA
  - svcB: nodeB
```